vuepress/README.md

```diff
@@ -60,6 +60,14 @@ footer: MIT Licensed | Copyright © 2020 kazuya kawaguchi
   <sponsor-button />
 </div>
 
+<div class="patreon" style="padding: 0 0 24px 0;">
+  <p style="text-align: center;">
+    <a href="https://www.patreon.com/kazupon" target="_blank">
+      <img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patreon">
+    </a>
+  </p>
+</div>
+
```

Comment:
I use GitHub Sponsors as my main patron services, so please remove the patreon. 🙏

Reply:
Hi @kazupon, but remove all occurrences of the patreon?

Reply:
@lucasferreiralimax 
Hmm, if you can remove them, i hope it. 🙏

Reply:
Yes, i remove in this [commit](https://github.com/kazupon/vue-i18n/pull/1104/commits/d94b193deb8c7307bcfba5a96fa8cf139f853cef) review please